app/models/song.rb

```diff
@@ -0,0 +1,4 @@
+class Song < ApplicationRecord
+  belongs_to :playlist
+  belongs_to :user
```

Comment:
oohhh good point. Thanks for catching that. I think what I need is to add to my ```User``` model:

 ```has_many :songs, :through :playlists```

right? I'm following [this example](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association)